<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QWidget" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>281</width>
     <height>115</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#e03e34;&quot;&gt;QTcp Client 客户端&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="ip"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="port">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>130</y>
     <width>390</width>
     <height>271</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>聊天记录:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="record">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>38</y>
      <width>372</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>405</y>
     <width>390</width>
     <height>71</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>49</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>输入框:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="msgEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>373</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>490</y>
     <width>491</width>
     <height>49</height>
    </rect>
   </property>
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBtn_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>读寄存器</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="linkBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getposBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>获取6维坐标</string>
   </property>
  </widget>
  <widget class="QLabel" name="strpos">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>pos:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moveBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>移动到</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_pos_X">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>210</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>220</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>260</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_pos_Y">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>300</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Z:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_pos_Z">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>290</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>RX:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_pos_RX">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>330</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_pos_RY">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>370</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>380</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>RY:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_pos_RZ">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>420</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>RZ:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>460</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>SP:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_pos_SP">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>450</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_1">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>modbustcp</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>normal_ascii</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>normal_rtu</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>kawasaki_json</string>
   </property>
  </widget>
  <widget class="QPushButton" name="laseropenBtn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string> 激光开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lasercloseBtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string> 激光关</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cameraopenBtn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>相机开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cameracloseBtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>40</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>相机关</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_tasknum">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>90</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>任务号:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tasknumwriteBtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>130</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>写入任务号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gitallinfoBtn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>130</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>获取信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
